src/components/calcite-link/calcite-link.tsx

```diff
@@ -93,6 +100,7 @@ export class CalciteLink {
       <Host role="presentation">
         <Tag
           class={{ [CSS_UTILITY.rtl]: dir === "rtl" }}
+          download={Tag === "a" && (download === "" || download) ? download : null}
```

Comment:
@jcfranco @eriklharper have you run into the following before...

- If a property is `boolean` or `string` and the property is set to `true`, the value is `""`?
- I had to work around this, I'm not sure if its as intended or stencil issue is needed?
- I had expected the value of `download` to be `true` if set to `true` via a property but it was a `""` empty string instead.
- Basically, I have to treat the value of `""` as `true` which seems counterintuitive and wrong 🤷

Reply:
When setting via a prop we also get this lovely warning.

```
$0.download = true;
true
index-07dce265.js:2915 stencil The state/prop "download" changed during rendering. This can potentially lead to infinite-loops and other bugs. 
Element <calcite-link href=​"calcite-link.html" role=​"presentation" calcite-hydrated download>​…​</calcite-link>​ 
New value  
Old value true
```

I guess stencil just doesn't like mixed boolean string types?

Reply:
Wow, interesting find. Can you add a comment there for posterity? 

I'll create a repro case and send it over to the Stencil team.